src/server/scripts/Kalimdor/BlackfathomDeeps/boss_kelris.cpp

```diff
@@ -47,10 +48,23 @@ class boss_kelris : public CreatureScript
     {
         boss_kelrisAI(Creature* creature) : BossAI(creature, DATA_KELRIS) { }
 
+        void Reset() override
+        {
+            _Reset();
+            DoCastSelf(SPELL_BLACKFATHOM_CHANNELING);
+        }
+
+        void JustReachedHome() override
+        {
+            _JustReachedHome();
+            DoCastSelf(SPELL_BLACKFATHOM_CHANNELING);
+        }
```

Comment:
```suggestion

```

Reply:
Very well, although I think this suggestion needs to include the blank line after the current code lines.